conf/broker.conf

```diff
@@ -37,8 +37,11 @@ webServicePortTls=8443
 # Enable the WebSocket API service in broker
 webSocketServiceEnabled=false
 
-# Control whether to bind directly on localhost rather than on normal hostname
-bindOnLocalhost=false
+# Hostname or IP address the service binds on, default is InetAddress.getLocalHost().getHostName().
+bindAddress=
```

Comment:
I guess that's a personal preference. Some people like things wide open by default, some not. I don't mind making it `0.0.0.0`.